swcregistrypython/test.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import json
+
+from unittest.mock import MagicMock
+from get_entry_info import SWC
+
+
+class TestMethods(unittest.TestCase):
+    def setUp(self):
+        with open('swc-definition.json') as f:
```

Comment:
A test can fail if swc-definition.json was changed

Reply:
What the alternatives are ?